book.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'rental'
+class Book
+  attr_accessor :title, author, rentals
+
```

Comment:
- [ ] Kindly make sure you correct the typing errors on the `attr_accessor` values to make it professional.
```ruby
# Bad
attr_accessor :title, author, rentals

# Good
attr_accessor :title, :author, :rentals
```